src/app/layout.tsx

```diff
@@ -0,0 +1,71 @@
+import type { Metadata, Viewport } from "next";
+import Script from "next/script";
+import "../styles/App.css";
+import "../styles/globals.css";
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <Script
+        async
+        src="https://umami.advisory.sg/script.js"
+        data-website-id="2f694eab-769e-420a-bb31-4efcb3662bf5"
+      />
+      <body>
+        <main>{children}</main>
+      </body>
+    </html>
+  );
+}
+
+export const viewport: Viewport = {
+  // See https://nextjs.org/docs/app/api-reference/functions/generate-viewport for configuration info
+  width: "device-width",
+  initialScale: 1,
+  themeColor: "#000000",
+};
+
+export const metadata: Metadata = {
+  // See https://nextjs.org/docs/app/api-reference/functions/generate-metadata for configuration info
+  title: "Advisory Mentorship Programme",
+  description: "The Advisory Mentorship Programme",
+
+  // Icons
+  icons: {
+    icon: [{ url: "/favicon.ico" }],
+    apple: [{ url: "/logo192.png" }],
+  },
+
+  // Manifest
+  manifest: "/manifest.json",
+
+  // TODO: Add Google metadata
+  // The following needs to be added to the HTML <head> somehow:
+  // <meta itemProp="name" content="Advisory Mentorship Programme" />
+  // <meta
+  //     itemProp="description"
+  //     content="The Advisory Mentorship Programme"
+  // />
+  // <meta itemProp="image" content="/advisory-mentorship-logo.jpg" />
```

Comment:
To address later on.

Reply:
addressed in 64480e00d11c839d0e98c0a6545a927ae1f3eae6